lib/business_time/config.rb

```diff
@@ -127,6 +134,25 @@ def weekdays
         end
       end
 
+      def load_holidays(holidays, container: config[:holidays], append: false, accept_parsing: false)
+        container.replace(DEFAULT_CONFIG[:holidays]) unless append
+        holidays.each do |holiday|
+          container.add(if Date === holiday
```

Comment:
In general, every time you find yourself writing `each { |elem| add(elem) }`, the God kills a kitten :)

`map-reduce` is called this way (and not `each-reduce` :) due to some reasons.